### PR TITLE
Create VA for shiftstack

### PR DESCRIPTION
**Pull request by rlobillo**
Create separated reproducer to include the needed changes to support openshift as workload on RHOSP18

As a pull request owner and reviewers, I checked that:
- [ ] Appropriate testing is done and actually running
- [ ] Appropriate documentation exists and/or is up-to-date:
- [ ] README in the role
- [ ] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: scenarios/reproducers/validated-architecture-shiftstack.yml [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+# Test Ceph file and object storage (block is enabled by default)
+cifmw_ceph_daemons_layout:
+  rgw_enabled: true
+  dashboard_enabled: false
+  cephfs_enabled: true
+
+# OCP on OSP needs to access OSP API from the VMs that behaves as OCP nodes
+cifmw_allow_vms_to_reach_osp_api: true
+# HCI requires bigger size to hold OCP on OSP disks
+cifmw_block_device_size: 100G
```
Comment (rlobillo) [inline scenarios/reproducers/validated-architecture-shiftstack.yml:159]: This is different from validated-architecture-1.yml

File: reproducer.yml [hunk 2]
```diff
@@ -55,3 +55,9 @@
   roles:
     - role: ci_setup
     - role: reproducer
+
+  post_tasks:
+    - name: Allow traffic from OSP VMs to OSP API (needed for shiftstack)
+      ansible.builtin.shell: iptables -I LIBVIRT_FWI 1 -o ocpbm -j ACCEPT
+      become: true
+      when: cifmw_allow_vms_to_reach_osp_api
```
Comment (rlobillo) [inline reproducer.yml:63]: This is needed for allowing the VMs to contact the OSP API, which is a requirement for the shiftstack OCP nodes to manage OSP resources that are created during the installation process and the openstack integration operators.

File: scenarios/reproducers/validated-architecture-shiftstack.yml [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+          ternary(4, 1)
+        }}
+      amount: "{{ [cifmw_libvirt_manager_compute_amount|int, 3] | max }}"
+      image_url: "{{ cifmw_discovered_image_url }}"
+      sha256_image_name: "{{ cifmw_discovered_hash }}"
+      image_local_dir: "{{ cifmw_basedir }}/images/"
+      disk_file_name: "base-os.qcow2"
+      disksize: 200
+      memory: 50
+      cpus: 8
```
[the remark this answers is not in the document]
Comment (psathyan) [reply scenarios/reproducers/validated-architecture-shiftstack.yml:93]: Is this something planned for virtual environments?

[the remark this answers is not in the document]
Comment (rlobillo) [reply scenarios/reproducers/validated-architecture-shiftstack.yml:93]: Yes! We want to explore shiftstack on virtual envs. We expect performance issues but it is useful for us as development environment, our automation gate and even bug verification.

**Comment by rlobillo** [comment]
wrong rebasing

**Comment by rlobillo** [comment]
replaced by https://github.com/openstack-k8s-operators/ci-framework/pull/1259

**Pull request closed without merging**